server.py

```diff
@@ -35,10 +35,14 @@ def validate_channel_name(chan: str) -> bool:
 
 def validate_nick(nick: str) -> bool:
     '''Enforce RFC 2812 nick name'''
-    # TODO: not strict enough
```

Comment:
This TODO should stay, since this still isn't quite RFC2812 (for example, you're not allowed to start a nick with numbers).